docker-compose.yml

```diff
@@ -73,7 +77,7 @@ services:
     volumes:
       - ${HOME}/.cloudflared:/etc/cloudflared
     environment:
-      - TUNNEL_URL=https://traefik.kitt.run
+      - TUNNEL_URL=https://${KITT_ARGO_URL}
       - TUNNEL_HOSTNAME=${KITT_ARGO_TUNNEL}
```

Comment:
Can we use KITT_TUNNEL_URL and KITT_TUNNEL_HOSTNAME as the env var names?

Reply:
I think TUNNEL_URL and TUNNEL_HOSTNAME is hardcoded in cloudflared
https://developers.cloudflare.com/argo-tunnel/reference/arguments/#url

Reply:
oh i see you want to remove ARGO from the env vars

Reply:
I just made them up for no good reason... should have used the CF env vars as a hint.